src/list.tsx

```diff
@@ -122,17 +122,19 @@ interface CommonProps {
     extra?: React.ReactNode;
     dataAttributes?: DataAttributes;
     disabled?: boolean;
+    navigable?: boolean;
```

Comment:
so, the idea of this prop is to only use it in conjunction with `renderRight` when we want the chevron to be rendered at the same time than the custom right element?

Reply:
Yep! That's right.

Reply:
I'd add a comment explaining this. If you use jsdoc, vscode will show the comment to users of this component